SeamCarver.js

```diff
@@ -1,18 +1,33 @@
 "use strict";
 
+const RED = 0;
+const GREEN = 1;
+const BLUE = 2;
+const BORDER_ENERGY = 1000;
+
 class SeamCarver {
     constructor(canvas) {
         this.width = canvas.width;
         this.height = canvas.height;
         this.picture = [];
```

Comment:
I think this could be made more efficient if we use [`Int16Array`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray)